docs/installation/server-config/index.rst

```diff
@@ -115,11 +111,10 @@ For more information, see :ref:`Streams parameter reference <streams_developer-g
 .. _ksql-commit-interval-ms:
 
 ^^^^^^^^^^^^^^^^^^
-commit.interval.ms
+ksql.streams.commit.interval.ms
 ^^^^^^^^^^^^^^^^^^
 
-The frequency to save the position of the processor. The default value in KSQL is ``2000``. Here is an example to change
-the value to ``5000`` by using the KSQL command line:
+The frequency to save the progress of a KSQL query.  The default value in KSQL is ``2000``, which means that KSQL will commit offsets (and thus mark progress) every 2 seconds.  Here is an example to commit offsets every 5 seconds:
```

Comment:
Should we document here what the implications of longer / shorter intervals are?